src/Workspaces/MSBuildTest/Resources/SolutionFiles/VisualBasic.sln

```diff
@@ -0,0 +1,20 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 11.00
+# Visual Studio 2010
```

Comment:
![image](https://user-images.githubusercontent.com/146967/196292359-de0fb584-ff8d-4f69-9a66-3d1b47cdf547.png)

Reply:
I copied and pasted the VB/C# mixed solution file and deleted the bits we didn't need. :smile: